### ISSUE TITLE
False positive when using extract-text-webpack-plugin

### ISSUE THREAD

**Issue by Milad** [depcheck/depcheck#237]
extract-text-webpack-plugin allows us to extract CSS code to a css file instead of depending on webpack to inject styles into the webpage.

My usage in webpack.config.js is the standard:
```javascript
            {
                test: /\.css$/,
                use: ExtractTextPlugin.extract({
                    fallback: 'style-loader',
                    use: 'css-loader'
                })
            },
            {
                test: /\.sass$/,
                use: ExtractTextPlugin.extract({
                    fallback: 'style-loader',
                    use: ['css-loader', 'sass-loader']
                })
            },
```
But depcheck claims they are not used, node-sass is a dependency of sass-loader and it will go if sass-loader is captured correctly.
```json
{
  "devDependencies": [
    "node-sass",
    "sass-loader",
    "style-loader"
  ]
}
```

**Comment by rumpl** [comment]
not stale
